rules/stylistic-issues.js

```diff
@@ -188,8 +188,19 @@ module.exports = {
         // require quotes around object literal property names
         "quote-props": ["error", "as-needed", { keywords: false, unnecessary: true, numbers: false }],
 
-        // require or disallow an empty line after variable declarations
-        "newline-after-var": ["error", "always"]
+        // enforce a maximum depth that blocks can be nested
+        "max-depth": ["error", 4],
```

Comment:
//cc @laurengarcia @MariagraziaAlastra

Reply:
:+1:

Might need to increase it a bit, pretty much all our code is in at least two blocks by default (e.g. IIFE, constructor).

Reply:
lgtm